playwright-e2e/dsm/component/tables/onc-history-table.ts

```diff
@@ -294,15 +303,15 @@ export default class OncHistoryTable extends Table {
 
   /* XPaths */
   private get tableXPath(): string {
-    return `//app-onc-history-detail//table[contains(@class,'table')]`
+    return `//app-onc-history-detail//table[contains(@class,"table")]`
   }
 
   private get rowXPath(): string {
     return '/tbody/tr'
   }
 
   private get lookupListXPath(): string {
-    return '//app-lookup/div/ul/li'
+    return '//ancestor-or-self::td//app-lookup/div/ul/li'
```

Comment:
The Lookup in an input or textarea field should have the same `td` as the root element.